main.nf

```diff
@@ -115,21 +115,23 @@ process odgiStats {
   path("${graph}.stats")
 
   """
-  odgi stats -i $graph -S -s -d -l > "${graph}.stats"
+  odgi stats -i $graph -S -s -d -l > "${graph}.stats" 2>&1
```

Comment:
Why do we need to mix both streams again?

Reply:
Because `odgi stats` has inconsistent implementation. The output of `-S` is printed to `stdout`, the output of `-s -d -l` to `stderr`. Else your MultiQC fails.
Maybe @andreaguarracino is also interested in this.

Reply:
Yeah. I would suggest to fix this in odgi stats.

Reply:
I expect this will be taken care of, maybe by me. But it will take some time until it is available for us here. We can live with the current suggestion.